cmake/OpenCVFindTimVX.cmake

```diff
@@ -0,0 +1,68 @@
+set(OPENCV_LIBTIMVX_ROOT_DIR "" CACHE PATH "Path to libtim-vx installation")
+set(TIMVX_SDK_DIR "" CACHE PATH "Path to Tim-VX SDK")
+# set(TIMVX_SDK_LIB_CANDIDATES "ArchModelSw;CLC;Emulator;GAL;NNArchPerf;OpenVXC;OpenVX;OpenVXU;vdtproxy;VSC" CACHE STRING "Tim-VX's SDK library candidates list")
+
+# function(find_timvx_sdk_libs _found)
+#     foreach(one ${TIMVX_SDK_LIB_CANDIDATES})
+#         find_library(TIMVX_${one}_LIBRARY ${one} PATHS "${TIMVX_SDK_DIR}/lib")
+#         if(TIMVX_${one}_LIBRARY)
+#             list(APPEND _sdk_libs ${TIMVX_${one}_LIBRARY})
+#         endif()
+#     endforeach()
+#     list(APPEND _sdk_libs ${TIMVX_SDK_DIR}/lib/libOpenVX.so.1)
+#     set(${_found} ${_sdk_libs} PARENT_SCOPE)
+# endfunction()
+
+set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -Wl,-rpath-link,\"${TIMVX_SDK_DIR}/lib\"")
```

Comment:
Compiling in this way does not require loading the TIM-VX's SDK libraries into `LD_LIBRARY_PATH`, but when running related programes still does. Those libraries are pre-compiled and shipped with TIM-VX source code. Maybe we need to contact devs from Verisilicon to do it in a better way.

Reply:
Building TIM-VX as static library allow us not to add `libtim-vx.so` to `LD_LIBRARY_PATH`. However, when running TIM-VX related programes, we still have to export `prebuilt-sdk/${platform}/lib` to `LD_LIBRARY_PATH`.